cypress/integration/ResultsPagerEnglish.feature

```diff
@@ -10,13 +10,17 @@ Feature: As a user, I would like to be able to page through search results from
     And both pagers display numbers 1 and 2, followed by "...", the last page number and the option to click "Next >" and "Go to Page" for screen readers
     And user clicks "Next" and displays 1, 2, and 3 followed by "...", and 2 highlighted as the page they are on and "Go to Page" for screen readers and 560 and "Next >"
     And the last page number to be visible
+    And the page offset is set to '/?swKeyword=tumor&page=3&pageunit=20&Offset=40'
```

Comment:
I thought #45 requires that the `Offset` parameter be removed completely.

Reply:
I fixed it mathematically - but didn't remove, wasn't 100% sure this was not needed yesterday..